src/libraries/RegistryHashes.sol

```diff
@@ -8,20 +8,16 @@ import {IDelegateRegistry} from "../IDelegateRegistry.sol";
  *
  * The encoding for the 5 types of delegate registry hashes should be as follows
  *
- * ALL:         keccak256(abi.encode(from, rights, to))
- * CONTRACT:    keccak256(abi.encode(from, rights, to, contract_))
- * ERC721:      keccak256(abi.encode(from, rights, to, tokenId, contract_))
- * ERC20:       keccak256(abi.encode(from, rights, to, contract_))
- * ERC1155:     keccak256(abi.encode(from, rights, to, tokenId, contract_))
+ * ALL:         keccak256(abi.encodePacked(rights, from, to))
+ * CONTRACT:    keccak256(abi.encodePacked(rights, from, to, contract_))
+ * ERC721:      keccak256(abi.encodePacked(rights, from, to, contract_, tokenId))
+ * ERC20:       keccak256(abi.encodePacked(rights, from, to, contract_))
+ * ERC1155:     keccak256(abi.encodePacked(rights, from, to, contract_, tokenId))
```

Comment:
We're using encode() instead of encodePacked() to prevent collision attacks

Reply:
It is collision safe as long as no dynamic length arguments (e.g. string, bytes) are used.